src/charm.py

```diff
@@ -335,14 +340,21 @@ def create_profile(self, auth_username, profile_name, resource_quota):
         except ApiError as e:
             self.log.info(f"Failed to create profile, error: {str(e)}")
         # TODO add resource quota to spec
+        # quota_dict=self._load_text_to_json(resource_quota)
```

Comment:
I suspect these comments can be removed, because we are passing `resource_quota` in already.